feature/media_details/presentation/src/main/java/com/sanaa/presentation/screen/episodeDetails/EpisodeDetailsScreenViewModel.kt

```diff
@@ -108,22 +121,20 @@ class EpisodeDetailsScreenViewModel @Inject constructor(
     override fun onSubmitRateBottomSheet() {
         tryToExecute(
             callee = ::submitEpisodeRating,
-            onError = ::onErrorAccrue
+            onError = { exception ->
```

Comment:
It was in a reference function; why did you change it?
```suggestion
            onError = ::onErrorAccrue
```